README.md

```diff
@@ -52,11 +52,6 @@ In short, SeaTechNerd83 and I combined the two scripts to run voice commands thr
 - **VoiceAttack**
   - [voiceattack.com](https://voiceattack.com)
 
-- **FFmpeg** (must be in your PATH)
```

Comment:
ffmpeg is now bundled in a Python library used by faster-whisper so doesn't need to be installed separately.